### ISSUE TITLE
inference / training only based on bounding boxes

### ISSUE THREAD

**Issue by otten456** [youngwanLEE/centermask2#24]
Hi,

according to the results in the paper you listed for the COCO dataset, centermask is under the current state-of-the-art architectures for object detection. For my use case I only have bounding box labeled training data. I'm wondering if it is possible to train centermask only on bounding boxes?

Hope you can give provide some information, thank you so much!

**Comment by youngwanLEE** [comment]
@otten456 

Unfortunately, our centermask needs the mask labels. 

If your case is only for object detection, I recommend [fcos](https://github.com/aim-uofa/AdelaiDet) that is the fundamental object detector we use.
